File: app/monitoring/pdfHealthCheck.scala
package monitoring

import app.ConfigProperties._
import gov.dwp.carers.CADSHealthCheck
import gov.dwp.carers.CADSHealthCheck.Result

class pdfHealthCheck extends CADSHealthCheck(s"${getProperty("application.name", default="p1")}", getProperty("application.version", default="x1").takeWhile(_ != '-')) {
  override protected def check: CADSHealthCheck.Result = {
    Result.healthy()
  }
}

File: src/main/java/monitoring/PdfHealthCheck.java
package monitoring;

import gov.dwp.carers.CADSHealthCheck;


/**
 * Created by peterwhitehead on 05/05/2016.
 */
public class PdfHealthCheck extends CADSHealthCheck {
    public PdfHealthCheck(String application, String version) {
        super(application, version);
    }

    @Override
    protected CADSHealthCheck.Result check() {
        return Result.healthy();
    }
}
